packages/main/test/pages/Simple.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
 <html>
-<head>
```

Comment:
Why is this file needed?

Reply:
I will reverted, only used it for testing purposes.